caddyhttp/httpserver/plugin.go

```diff
@@ -340,6 +343,15 @@ func standardizeAddress(str string) (Address, error) {
 		}
 	}
 
+	// convert non-ascii (IDNA) host to Punycode
+	if isIDN(host) {
+		asciiHost, err := idna.ToASCII(host)
+		if err != nil {
+			return Address{}, fmt.Errorf("Failed to convert IDN to ASCII: %s", input)
```

Comment:
Error strings should start with a lower-case character (lint). Change to `"failed to ..."`